README.md

```diff
@@ -4,10 +4,20 @@ A gRPC interceptor to mask errors.
 
 This piece of middleware intends to stop sensitive information from being returned to calling services. Pass in a list of codes you wish to mask, and they shall all be returned as internal server errors, with no further information.
```

Comment:
I believe this is missing a bit about remapping the functionality. Maybe the one-liner should include a reference to the remapping as well. Or possibly it could be turned into something more holistic about masking errors.